src/routing_table/network_tests.rs

```diff
@@ -309,33 +309,39 @@ pub fn verify_network_invariant<'a, T, U>(nodes: U)
         node.verify_invariant();
     }
     // check that prefixes are disjoint
-    verify_disjoint_prefixes(groups.keys().into_iter().cloned().collect());
+    for prefix1 in groups.keys() {
+        for prefix2 in groups.keys() {
+            if prefix1 == prefix2 {
+                continue;
+            }
+            if prefix1.is_compatible(prefix2) {
+                panic!("Group prefixes should be disjoint, but these are not:\n\
+                    Group {:?}: {:?}\n\
+                    Group {:?}: {:?}",
+                       prefix1,
+                       groups[prefix1].1,
+                       prefix2,
+                       groups[prefix2].1);
```

Comment:
This comment addresses different lines of code; these have been changed to avoid printing the same name twice.